src/entities/message-template/message-template.ts

```diff
@@ -221,6 +221,30 @@ export class MessageTemplate extends UniverseEntity<MessageTemplatePayload, Mess
     return new MessageTemplate({ rawPayload: payload, universe, http, initialized: true })
   }
 
+  public async post (): Promise<Entity<MessageTemplatePayload, MessageTemplateRawPayload>> {
+    try {
+      return await this._post()
+    } catch (e: any) {
+      if (e?.properties?.error?.response?.status === 422) {
+        throw new EntityPostError(e?.properties?.error?.response?.data?.msg, { error: e })
```

Comment:
@eljefedelrodeodeljefe @haayhappen not really sure if this is the right practice here. Please let me know if there is a better way here